presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -246,6 +246,11 @@ Accessors
     for single geometries returns 1,
     for empty geometries returns 0.
 
+.. function:: ST_Geometries(Geometry) -> Geometry
+
+   Returns an array of geometries in the specified collection. Returns null if input
```

Comment:
Let's clarify that this function return a one-element array if input geometry is not a multi-geometry.